src/Markup/Avalonia.Markup.Xaml/Converters/MemberSelectorTypeConverter.cs

```diff
@@ -18,10 +18,7 @@ public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceT
 
         public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
         {
-            return new MemberSelector
-            {
-                MemberName = (string)value,
-            };
+            return MemberSelector.Parse((string)value);
```

Comment:
Any reason that this type converter can't hook into the `ParseTypeConverter` logic?

Reply:
It's because this type converter is needed on an interface: `IMemberSelector`, so the `ParseTypeConverter` can't locate `MemberSelector.Parse` as it doesn't know that `MemberSelector` provides the `Parse` implementation.

Reply:
Sounds good.